bin/docker_run.sh

```diff
@@ -12,4 +12,9 @@ if [ $? -ne 0 ]; then
     schematic migrations/
 fi
 
+if [ "$BRAINTREE_MERCHANT_ID" ]; then
```

Comment:
I think, -z is "True of the length if "STRING" is zero." - http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html

```
pay master payments-service $ cat foo.bash 
if [ "$FOO" ]; then
    echo "Found env."
fi
pay master payments-service $ env | grep FOO
pay master payments-service $ ./foo.bash 
pay master payments-service $ export FOO=
pay master payments-service $ ./foo.bash 
pay master payments-service $ export FOO=something
pay master payments-service $ ./foo.bash 
Found env.
```